features/options/contract/version.feature

```diff
@@ -6,13 +6,6 @@ Feature: osrm-contract command line options: version
 
     Background:
         Given the profile "testbot"
-    
-    Scenario: osrm-contract - Version, short
```

Comment:
Good catch, strictly speaking this is a breaking change and results in us having to wait till 6.0 until we can ship this. Probably makes more sense to search for a different mnemonic.